### PR TITLE
feat: Adding cubi-tk snappy itransfer_sv_calling

### PR DESCRIPTION
**Pull request by Nicolai-vKuegelgen**
Adding a new command to transfer all outout from snappy's sv_calling or sv_calling_targeted steps.
Includes autodetection of which step was run and autodetection of all used sv callers from snappy config.

### CODE REVIEW

File: cubi_tk/snappy/itransfer_sv_calling.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+from . import common
+from .itransfer_common import IndexLibrariesOnlyMixin, SnappyItransferCommandBase
+
+#: Template string for variant_calling results files.
+TPL_INPUT_DIR = "%(step_name)s/output/%(mapper)s.%(caller)s.%(library_name)s"
+
+
+class SnappyStepNotFoundException(Exception):
+    def __str__(self):
+        return "snappy-pipeline is not installed. This function will not work."
```
Comment (sellth) [inline cubi_tk/snappy/itransfer_sv_calling.py:18]: Adapt this

**Review by sellth — CHANGES_REQUESTED** [review]

File: cubi_tk/snappy/common.py [hunk 2]
```diff
@@ -16,6 +16,18 @@ class CouldNotFindBioMedSheet(Exception):
...
+def load_snappy_config(path_yaml):
+    """Load SNAPPY configuration file.
+
+    :param path_yaml: Path to SNAPPY configuration file.
+    :type path_yaml: pathlib.Path
+
+    :return: Returns configuration dictionary.
+    """
+    with open(path_yaml, "rt") as f:
+        return yaml.safe_load(f)
```
[the remark this answers is not in the document]
Comment (Nicolai-vKuegelgen) [reply cubi_tk/snappy/common.py]: agreed

File: cubi_tk/snappy/itransfer_sv_calling.py (hunk 1 above)
Comment (Nicolai-vKuegelgen) [reply cubi_tk/snappy/itransfer_sv_calling.py]: done

**Review by sellth — APPROVED** [review]
LGTM

**Pull request merged**